src/components/data/powerArrayHolders.js

```diff
@@ -0,0 +1,24 @@
+const powerArrayHolders = {
```

Comment:
A bit unsure of how these arrays factor into the implementation. What are these necessary for, and will they change the charsheet data, and how we will be getting powers for the PDF?